doc/user/flycheck-versus-flymake.rst

```diff
@@ -44,71 +37,68 @@ Please do **not** use this table alone to make your personal judgment.  Read the
+|`Automatic syntax checker  |by major mode and      |user-defined [#]_      |
+|selection <Syntax checker  |custom predicates      |                       |
+|selection_>`_              |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Manual syntax checker     |yes                    |yes                    |
+|selection <Manual          |                       |                       |
+|Selection_>`_              |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Multiple syntax checkers  |yes                    |yes                    |
+|per buffer`_               |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Supported languages`_     |>50                    |2 [#]_                 |
+|                           |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Checking remote files      |said to work, but not  |partly?                |
+|via Tramp                  |officially supported   |                       |
+|                           |[#]_                   |                       |
++---------------------------+-----------------------+-----------------------+
+|`Definition of new         |single declarative     |function definition    |
+|syntax checkers`_          |function/macro         |[#]_                   |
++---------------------------+-----------------------+-----------------------+
+|`Error levels`_            |errors, warnings,      |errors, warnings,      |
+|                           |infos, custom levels   |notes, custom levels   |
+|                           |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Error identifiers`_       |yes                    |no                     |
++---------------------------+-----------------------+-----------------------+
+|`Error parsing`_           |can use arbitrary      |can use arbitrary      |
+|                           |functions; helpers for |parsing functions; no  |
+|                           |regexp, JSON and XML   |built-in helpers       |
++---------------------------+-----------------------+-----------------------+
+|Error highlighting in      |yes                    |yes                    |
+|buffers                    |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Fringe icons for errors    |yes                    |yes (Emacs 24.1+)      |
++---------------------------+-----------------------+-----------------------+
+|`Error message display`_   |tooltip, echo area,    |tooltip only [#]_      |
```

Comment:
Wrong on flymake, but I will get back to you later on on exactly what it does.

Reply:
@joaotavora I think that's the only remaining point that needs to be updated.  Can you elaborate on what Flymake offers for error display (default behavior, built-in customization)?

Please also check the [related section](http://www.flycheck.org/en/1403-update-flymake-comparison/user/flycheck-versus-flymake.html#error-message-display) which is probably out of date.